### PR TITLE
Roe 278 send roe bo data

### PR DESCRIPTION
**Pull request by mattch1**
https://companieshouse.atlassian.net/browse/ROE-278

Updated model with:

beneficial owners statement
beneficial owners individual
beneficial owners corporate
beneficial owners government or public authority

### CODE REVIEW

**Review by markpit — APPROVED** [review]

File: src/services/overseas-entities/types.ts [hunk 1]
```diff
@@ -5,6 +5,10 @@
 export interface OverseasEntity {
     presenter?: Presenter;
     entity?: Entity;
+    beneficial_owners_statement?: BeneficialOwnersStatement;
```
Comment (mouhajer-ch) [inline src/services/overseas-entities/types.ts:8]: The `beneficial_owners_statement` type should be of type `BeneficialOwnersStatementType` not `BeneficialOwnersStatement`

Comment (markpit) [reply src/services/overseas-entities/types.ts:8]: i think that is my fault as that is what is in the api spec, i've included it as an object, perhaps you are right and teh spec should change but I think this PR matches the current spec - https://github.com/companieshouse/overseas-entities-api/blob/main/spec/schema.json#L153

Comment (mattch1) [reply src/services/overseas-entities/types.ts:8]: Shouldn't it be:
Object name: BeneficialOwnersStatement
Field name: BeneficialOwnersStatementType

Comment (mouhajer-ch) [reply src/services/overseas-entities/types.ts:8]: Like this
```
export interface OverseasEntity {
    ...
    beneficial_owners_statement?: BeneficialOwnersStatementType;
    ...
}
```

**Review by mouhajer-ch — APPROVED** [review]

**Pull request merged**